src/VsTestResults.trx

```diff
@@ -0,0 +1,812 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<TestRun id="140d787a-23c7-478d-b9c8-eeb344604aa8" name="shasnayak@DESKTOP-V5VHPHF 2021-06-15 18:56:36" runUser="NORTHAMERICA\shasnayak" xmlns="http://microsoft.com/schemas/VisualStudio/TeamTest/2010">
+  <Times creation="2021-06-15T18:56:36.4935878-04:00" queuing="2021-06-15T18:56:36.4935878-04:00" start="2021-06-15T18:56:23.2865498-04:00" finish="2021-06-15T18:56:36.6940195-04:00" />
```

Comment:
Is this the run result file?  Should we not check this in?

Reply:
Deleted this